zipline/country.py

```diff
@@ -0,0 +1,14 @@
+"""Canonical definitions of country code constants.
+"""
+from iso3166 import countries_by_name
+
+
+class CountryCode(object):
```

Comment:
I found this useful for ones where I didn't know the name offhand, or where the name was surprising to me. In particular, I kept wanting to use `'UK'` instead of `'GB'`. Going through iso3166 seemed like a nice way to make sure we're actually using the correct code for any given country.